### DISCUSSION TITLE
why the node registers with the poet in the current epoch, offline poet in the next epoch, and registers with the poe again in the next epoch?

### DISCUSSION THREAD

**Discussion by honglouzzc** [spacemeshos/go-spacemesh#6005] [Q&A]
@

**Comment by poszu** [comment]
@honglouzzc I don't fully understand the question. A poet round takes around an epoch (13.5 days). A node registers in poet in epoch N ("current" epoch), waits for 13.5 days, gets the proof (it's already epoch N+1), and publishes an ATX. The N+1 epoch is called a "publish" epoch.

**Reply by honglouzzc** [reply]
The same node gets the reward at different intervals

**Reply by poszu** [reply]
I still don't understand what this is about, sorry.

**Reply by honglouzzc** [reply]
> I still don't understand what this is about, sorry.

The rule for a node that receives a reward is: Epoch 18 20 22 24, but Epoch 19 21 23 does not receive any reward.
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 the config for start node like:
#!/bin/bash
#
cd /opt/1314kdy/spacemesh/sda
# 
sudo nohup ./go-spacemesh --listen /ip4/0.0.0.0/tcp/6001 --config config.mainnet.json --smeshing-start --smeshing-coinbase sm1XXXXXXXXXXXXX --smeshing-opts-numunits 100 --smeshing-opts-provider 4294967295 --smeshing-opts-datadir /mnt/sda/post_data --data-folder /mnt/sda/node_data --filelock=/tmp/spacemesh-sda.lock > msh.log &

**Reply by poszu** [reply]
OK, so the problem is that your node doesn't get rewards every other (apparently odd) epoch. There might be various reasons for this. Most likely it failed to publish an ATX targeting these epochs. Did you check if it published ATXs?

**Reply by honglouzzc** [reply]
> OK, so the problem is that your node doesn't get rewards every other (apparently odd) epoch. There might be various reasons for this. Most likely it failed to publish an ATX targeting these epochs. Did you check if it published ATXs?

did not published ATXs; 
node status is :
postStart,
postComplete,
postServiceStarted 

or  like follow status:
+++++++++++++++++++++++++++++++++++
  "timestamp": "2024-06-24T09:20:37.369585931Z",
  "help": "Node is waiting for PoET registration window in current epoch to open. After this it will submit challenge and wait until PoET round ends in publish epoch.",
  "poetWaitRound": {
    "current": 25,
    "publish": 26,
    "wait": "1201187.988486531s",
    "until": "2024-07-08T07:00:25.358067829Z",
    "smesher": "cSukU5tLDzuOtJM+Oxb3aHC7PGltAmYzsx4VhOR5oL0="
  }
}
+++++++++++++++++++++++++++++++++++++

**Reply by poszu** [reply]
It could have been too late to submit to poets. Perhaps PoST proving took too long?

**Reply by honglouzzc** [reply]
> It could have been too late to submit to poets. Perhaps PoST proving took too long?

Is it possible that it is a configuration file problem?  All nodes in any epoch, without human intervention, submit to poets。
Too strange...

**Reply by poszu** [reply]
I can't tell, unfortunately.  There could be many reasons.
